File: api/airTicket/src/main/java/com/xiang/airTicket/entity/Seat.java
package com.xiang.airTicket.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

// 座位表
@Entity
public class Seat {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String number; // 编号

    private Integer line; // 行数

    private Integer column; // 列数

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getNumber() {
        return number;
    }

    public void setNumber(String number) {
        this.number = number;
    }


    public Integer getLine() {
        return line;
    }

    public void setLine(Integer line) {
        this.line = line;
    }

    public Integer getColumn() {
        return column;
    }

    public void setColumn(Integer column) {
        this.column = column;
    }
}

File: webapp/airTicket/src/app/entity/Seat.ts
// 座位实体

import {ShipSpace} from './ShipSpace';

export class Seat {

  id: number;

  number: string; // 编号

  line: number; // 行数

  col: number; // 列数

  shipSpace: ShipSpace;

  constructor(id: number, number1: string, row: number, column: number) {
    this.id = id;
    this.number = number1;
    this.line = row;
    this.col = column;
  }
}
